gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/installation/OrganizationResource.java

```diff
@@ -89,4 +90,9 @@ public EndpointsResource getEndpointsResource() {
     public EntrypointsResource getOrganizationEntrypointsResource() {
         return resourceContext.getResource(EntrypointsResource.class);
     }
+
+    @Path("/plugins/policies")
```

Comment:
Apparently not in that case because the open-api already defines 2 baseURL (with or without the organizationId). (Seen with @phiz71 on my previous PR)